library/kani_core/src/mem_init.rs

```diff
@@ -119,6 +127,21 @@ macro_rules! kani_mem_init {
                         self.tracked_offset += to_offset - from_offset;
                         // Note that this preserves the value.
                     }
+                } else {
+                    self.bless::<LAYOUT_SIZE>(to_ptr, 1);
+                }
+            }
+
+            #[kanitool::disable_checks(pointer)]
+            pub fn bless<const LAYOUT_SIZE: usize>(&mut self, ptr: *const u8, num_elts: usize) {
```

Comment:
Please add comment. This seems to be assuming that all bytes are initialized, is that the case?

Reply:
Yes, correct. Will do!